File: level1/6 - Structures avancées/3 - Étape la plus longue.c
//coded by cybereagle2001
#include <stdio.h>
#define repeat(nb) for (int _loop=1, _max=nb; _loop<= _max;_loop++)

int main()
{
   int nbDeJours;
   int max1=0;
   scanf("%d",&nbDeJours);
   repeat(nbDeJours)
   {
      int distance;
      scanf("%d",&distance);
      if (distance> max1)
      {
         max1= distance;
      }
   }
   printf("%d",max1);
}

File: java/level 1/6- Le grand voyage/3- Étape la plus longue.java
/** coded by cybereagle2001*/

import algorea.Scanner;

class Main{
   static Scanner input= new Scanner(System.in);
   public static void main(String[] args){
      int number= input.nextInt();
      int max=0,dist;
      for (int loop=0; loop<number; loop++){
         dist= input.nextInt();
         if (dist > max){
            max= dist;
         }
      }
      System.out.println(max);
   }
}
